drivers/adodb-pdo_mysql.inc.php

```diff
@@ -19,7 +19,7 @@
  * @copyright 2014 Damien Regad, Mark Newnham and the ADOdb community
  */
 
-class ADODB_pdo_mysql extends ADODB_pdo {
+final class ADODB_pdo_mysql extends ADODB_pdo {
```

Comment:
Why `final` ?

Reply:
Thats a very open question - we use them a lot because we've thousands of classes in our application in its a just a useful way to track the usage so it's become a bit of a habit.  

Not sure why we wouldn't start setting these.  The way that we use classes is not very intuitive ( and not  well documented)

Reply:
Personally I see no added value in making our classes final... In fact, we even advertise customizing ADOdb by extending drivers [in the documentation](https://adodb.org/dokuwiki/doku.php?id=v5:userguide:learn_extensions:new_driver)... Why should we prevent someone from doing this with the PDO drivers ?